### PR TITLE
Leaderboard with API

### PR DESCRIPTION
**Pull request by RafaelEchart**
In this milestone I added the following features:

- Create a new game in the [API: Leaderboard](https://www.notion.so/Leaderboard-API-service-24c0c3c116974ac49488d4eb0267ade3)

QQQQQQQQQQ

- **With the created ID** The leaderboard app can now **POST** a new score of a player (validation input included)
- **With the created ID** the leaderboard app can now **GET** the saved scores from the API at the start of the app or when the refresh button is clicked
-The API request are made by **ASYNC AWAIT** and **TRY CATCH**
- Added some styles but I am saving the best for the last milestone :) 

Steps for running the application: 

- run the command "npm install"
- run the command "npm run build"
- run the command "npm run start"
- Open your browser in http://localhost:8080/ or the port that says the CLI

image

**THIS APPLICATION NEEDS INTERNET CONECTION**

_Hope you like it!_

### CODE REVIEW

File: dist/index.html [hunk 1]
```diff
@@ -13,11 +13,16 @@ <h1 class="main_title">Leaderboard app</h1>
 
       <div class="score-leaderboard">
```
Comment (SajjadAhmad14) [inline dist/index.html:14]: Please add project descriptions and instructions to run the project locally for a non-technical user in the README file

File: src/getScores.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    const leaderboardAPIrequest = await fetch(
+      'https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/afMNT2NsfBzm4rDRXx1A/scores/',
+      {
+        method: 'GET',
+        headers: {
+          'Content-Type': 'application/json',
+        },
+      },
+    );
+
```
Comment (SajjadAhmad14) [inline src/getScores.js:16]: The app does not function as expected. The user is unable to submit data and the score list does not change on clicking the refresh button. Please make sure the app is working as expected.

**Review by SajjadAhmad14 — CHANGES_REQUESTED** [review]
## Required Changes ♻️
> "Change is inevitable. Change is constant. -Benjamin Disraeli"
## Hi Rafael, 
 ### You have been doing a tremendous job so far :ok_hand:

## To Highlight 🎉
PR has a descriptive title and summary. :heavy_check_mark:
GitHub flow is followed. :heavy_check_mark:
## To Improve 🙏

- [ ] Please check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Please leave any questions or comments in the PR if you have any doubts or unclarity!

**_Note: Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: src/getScores.js (hunk 2 above)
Comment (RafaelEchart) [reply src/getScores.js:16]: What is the message you get, In my computer its working perfectly

PS: inputs cannot be empty

File: dist/index.html (hunk 1 above)
Comment (RafaelEchart) [reply dist/index.html:14]: Thanks! Working on this!

**Review by SajjadAhmad14 — APPROVED** [review]
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:

### Hi Rafael, 
## Congratulations 🎉 Thank you for implementing changes suggested previously 🙏

## Happy Coding 👏👏👏
#### Please leave any questions or comments in the PR if you have any doubts or unclarity!
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**